### PR TITLE
[DynamoDB] Added jacoco and tested control plane operations

### LINKED ISSUE #185: Add JaCoCo to DynamoDB plugin

### PR DESCRIPTION
**Pull request by amcp**
Fixes #185 and #187

### CODE REVIEW

File: ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/NdbenchAWSCredentialProvider.java [hunk 1]
```diff
@@ -37,11 +36,11 @@
 @Singleton
 public class NdbenchAWSCredentialProvider implements AWSCredentialsProvider {
 
-    private DynamoDBConfigs config;
+    private IDynamoDBConfiguration config;
```
Comment (ipapapa) [inline ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/NdbenchAWSCredentialProvider.java]: I believe that  Hungarian style notation has been deprecated.

Comment (amcp) [reply ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/NdbenchAWSCredentialProvider.java]: I saw an example of it in other places in ndbench. if you want me to revert, I will.

Comment (amcp) [reply ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/NdbenchAWSCredentialProvider.java]: for example https://github.com/Netflix/ndbench/blob/master/ndbench-janusgraph-plugins/src/main/java/com/netflix/ndbench/plugin/janusgraph/configs/IJanusGraphConfig.java

Comment (ipapapa) [reply ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/NdbenchAWSCredentialProvider.java]: That is a good point. I suggest we move towards the correct way. If you want to change that configuration to the proper notation in this PR, I am fine merging it. All the rest look good.

Comment (amcp) [reply ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/NdbenchAWSCredentialProvider.java]: I think we can change the notation in a different PR. Ill revert on this one

Comment (amcp) [reply ndbench-dynamodb-plugins/src/main/java/com/netflix/ndbench/plugin/dynamodb/NdbenchAWSCredentialProvider.java]: fixed

**Pull request merged**